source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -2336,6 +2340,8 @@
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_PREPROCESSOR_DEFINITIONS = "USEFLUENTUI=1";
+				"GCC_PREPROCESSOR_DEFINITIONS[arch=*]" = "";
```

Comment:
won't these 2 lines simply counteract each other?

Reply:
it's xcode gui doing.